docs/web/supported-browsers-and-frameworks.mdx

```diff
@@ -21,6 +21,90 @@ However, there are some gotchas to be aware of as well as common errors that you
 your stack.  The issues vary depending on the tooling in your project.  This page explains some helpful workarounds
 with links to examples.
 
+## Fetch API `keepalive`
```

Comment:
@rakuista In talking with @timostamm, we would like to move this information to the official Connect docs instead. I can put up a PR that addresses what Timo mentioned above, so let's just revert `docs/web/supported-browsers-and-frameworks.mdx` altogether. Thanks!

Reply:
Done.